src/SFA.DAS.AdminService.Web.Tests/Services/Gateway/OrganisationChecks/Orchestrator/OrganisationRiskTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Models;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.Api.Types.Models.UKRLP;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Services.Gateway.OrganisationChecks.Orchestrator
+{
+    [TestFixture]
+    public class OrganisationRiskTests
+    {
+        private GatewayOrganisationChecksOrchestrator _orchestrator;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<ILogger<GatewayOrganisationChecksOrchestrator>> _logger;
+
+        private static string ukprn = "10026709";
+        private static string UKRLPLegalName = "EVA WOMENS AID LTD.";
+        private static string UserName = "GatewayUser";
+
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _logger = new Mock<ILogger<GatewayOrganisationChecksOrchestrator>>();
+            _orchestrator = new GatewayOrganisationChecksOrchestrator(_applyApiClient.Object, _logger.Object);
+        }
+
+        [TestCase("Company and charity", "Eva Training and Consultancy")]
+        [TestCase(null, "Eva Training and Consultancy")]
+        [TestCase("Company and charity", null)]
+        [TestCase(null, null)]
+        public void Check_orchestrator_builds_with_organisation_risk(string organisationType, string tradingName)
```

Comment:
Can you remove the technical detail from the test name?

Reply:
Renaming is done.